asset/css/markbind.css

```diff
@@ -203,3 +203,7 @@ so we apply temporary styles to fix the layout until Vue correctly updates the D
 li.footnote-item:target {
     background-color: #eee;
 }
+
+.footnote-item-popover {
+    text-decoration: none !important;
```

Comment:
You mentioned that `!important` is not needed anymore since MarkBind/vue-strap#112 would fix the issue.

So please update this PR to remove it.

Reply:
It's removed already!

Reply:
My bad, I mistaken viewing your first commit for viewing final file changes.

Since the `!important` commit is part of the original solution, you should squash them into one.

Reply:
Ok, squashed the commits.